src/service_aggregator.py

```diff
@@ -165,7 +186,7 @@ def is_end_message(message):
     return False
 
 
-async def post_with_callback(host_url, query, guid, params=None):
+async def post_with_callback(host_url, query, guid, params):
```

Comment:
Why did this change from keyword arg to required? will this break anything?

Reply:
I changed this to an empty dict and removed the POST logic below this. POSTing with params doesn't actually work.